view/layout.dust

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+      <title>{title}</title>
+      <link href="https://static.parliament.uk/pugin/1.9.4/stylesheets/main.css" rel="stylesheet">
```

Comment:
can dust use 'real' javascript? Or can we make the pugin version come from the app.js file maybe?